docs/generated/redact_safe.md

```diff
@@ -10,6 +10,9 @@ pkg/roachpb/metadata.go | `RangeID`
 pkg/roachpb/metadata.go | `ReplicaID`
 pkg/roachpb/metadata.go | `RangeGeneration`
 pkg/roachpb/metadata.go | `ReplicaType`
+pkg/sql/catalog/descpb/structured.go | `ID`
```

Comment:
Thanks. Is it possible to also add some of the other descriptor IDs in `descpb/structured.go` (`ColumnID`, `IndexID`, `FamilyID`, `MutationID`)?

Reply:
taking note, will do

Reply:
Thanks. I think we want `descpb.DescriptorVersion` as well.